packages/@react-spectrum/combobox/docs/ComboBox.mdx

```diff
@@ -87,7 +87,7 @@ function Example() {
         label="Pick an engineering major"
         defaultItems={options}
         onSelectionChange={setMajorId}>
-        {item => <Item>{item.name}</Item>}
+        {item => <Item key={item.name}>{item.name}</Item>}
```

Comment:
I don't think I've seen the key warning you referenced [in the wild yet](https://codesandbox.io/s/bitter-river-57sjgg?file=/src/App.js), do you happen to have a reproduction? Omitting the key in favor of having an `id` in the option list is what this example is supposed to portray so ideally we'd keep this example at the very least

Reply:
@LFDanLu I was super wrong about the need for this. Sorry, I'd missed that in the absence of a key attribute, the code used the item.id! This whole thing can be closed, thanks for the input.